spec/index.html

```diff
@@ -260,9 +260,14 @@ <h2>Notation and Terminology</h2>
 
     <p>A <dfn>ground</dfn> RDF graph is one that contains no blank nodes.</p>
 
+    <p>
+    An RDF term <dfn>appears in</dfn> a graph G if it is in the set TM(G), defined inductively as follows: (1) for each triple in the graph G, its subject, predicate, and object are in TM(G); (2) for each triple term in TH(G), its subject, predicate, and object are in TM(G).<br/>
+    A triple <b><i>appears in</i></b> a graph G if it is in G or if the triple term composed by its subject, predicate, and object appears in G.
```

Comment:
I've not changed any text here, only white space in the source, where shorter lines are far easier to review. Also, if it's worth `<br/>`, it's usually worth `<br/><br/>`. Also also, I'm wondering why the `(1)(2)` don't warrant `<br/>` or `<ol>`.
```suggestion
    An RDF term <dfn>appears in</dfn> a graph G if it is in the set TM(G),
    defined inductively as follows:
    (1) for each triple in the graph G,
        its subject, predicate, and object are in TM(G);
    (2) for each triple term in TH(G),
        its subject, predicate, and object are in TM(G).
    <br/><br/>
    A triple <b><i>appears in</i></b> a graph G
    if it is in G or
    if the triple term composed by its subject, predicate, and object appears in G.
```